strings-csharp/GenerateStreamFromStringInCSharp/GenerateStreamFromStringInCSharp/GenerateStreamFromStringMethods.cs

```diff
@@ -10,7 +10,7 @@ public static Stream
             encoding ??= Encoding.UTF8;
 
             var stream = new MemoryStream(encoding.GetByteCount(sampleString));
-            var writer = new StreamWriter(stream, encoding);
+            var writer = new StreamWriter(stream, encoding, -1);
```

Comment:
This is still missing the using statement and we need to explicitly call the `StreamWriter` overload that leaves the stream open.
https://learn.microsoft.com/en-us/dotnet/api/system.io.streamwriter.-ctor?view=net-7.0#system-io-streamwriter-ctor(system-io-stream-system-text-encoding-system-int32-system-boolean)